src/test/resources/eager/reconstructs-with-multiple-loops.expected.jinja

```diff
@@ -0,0 +1,23 @@
+[][]
+[0][0]
+{% set alpha,beta = [0],[0] %}{% for i in deferred %}
+  {% if deferred %}
+    {% for j in deferred %}
+      {% if deferred %}
+        {% do alpha.append(1) %}
+      {% else %}
+        {% do alpha.append(2) %}
+      {% endif %}
+    {% endfor %}
+  {% else %}
+    {% for j in deferred %}
+      {% do beta.append(1) %}
+      {% if deferred %}
+        {% do alpha.append(3) %}
+      {% else %}
+        {% do alpha.append(4) %}
+      {% endif %}
+    {% endfor %}
+  {% endif %}
+{% endfor %}
+{{ alpha ~ beta }}
```

Comment:
Before this change, the output would look like:
```
[][]
[0][0]
{% set beta = [0, 1] %}{% for i in deferred %}
  {% if deferred %}
    {% set alpha = [0] %}{% for j in deferred %}
      {% if deferred %}
        {% do alpha.append(1) %}
      {% else %}
        {% do alpha.append(2) %}
      {% endif %}
    {% endfor %}
  {% else %}
    {% set alpha = [0] %}{% for j in deferred %}
      {% do beta.append(1) %}
      {% if deferred %}
        {% do alpha.append(3) %}
      {% else %}
        {% do alpha.append(4) %}
      {% endif %}
    {% endfor %}
  {% endif %}
{% endfor %}
{{ [0] ~ beta }}
```
Clearly this is wrong for 2 reasons: `alpha` does not get deferred (and its value is reset each for loop to `[0]`), and `beta` starts off with the wrong value